man/systemd.network.xml

```diff
@@ -872,6 +872,17 @@ Table=1234</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>IPv6RetransTime=</varname></term>
+        <listitem>
+          <para>Configures IPv6 Retransmission Time. Takes a 32-bit unsigned integer.  The
+          time, in milliseconds, between retransmitted Neighbor Solicitation messages.  Used
+          by address resolution and the Neighbor Unreachability Detection algorithm.</para>
```

Comment:
hmm, as i read the patch: if you set this to zero, the kernel default is left in place here. this should be documented here.

Reply:
The value of zero means use the current value, which may be different to the kernel default. I'll add some text.